### PR TITLE
[easy] Fix documentation for ImageStack.sel

### PR DESCRIPTION
**Pull request by ttung**
screen shot 2018-12-19 at 5 39 19 pm

### CODE REVIEW

**Comment by ttung** [comment]
Without the indentation, it looks like:

screen shot 2018-12-19 at 5 40 52 pm

File: starfish/imagestack/imagestack.py [hunk 1]
```diff
@@ -335,19 +335,19 @@ def sel(self, indexers: Mapping[Indices, Union[int]]):
         Examples
         --------
 
-        Create an Imagestack using the ``synthetic_stack`` method::
```
Comment (joshmoore) [inline starfish/imagestack/imagestack.py]: Looks like we have a mix of ending these Example introductions with `::`, :`:, and no colon as here.

Comment (ttung) [reply starfish/imagestack/imagestack.py]: Can someone explain what the differences in all these might be?  Hard to google this sort of thing. :(

Comment (joshmoore) [reply starfish/imagestack/imagestack.py]: Yeah, I don't know. Yours certainly looked ok. But `::` did as well. I didn't look up what `:` came out looking like. Probably just best if we pick one we like and all agree on it.

**Pull request merged**